src/DotLiquid.Tests/StandardFilterTests.cs

```diff
@@ -1654,6 +1654,7 @@ public void TestReverse()
             CollectionAssert.AreEqual(arrayReversed, StandardFilters.Reverse(array));
             CollectionAssert.AreEqual(array, StandardFilters.Reverse(arrayReversed));
             CollectionAssert.AreEqual(new[] { 3, 2, 2, 1 }, StandardFilters.Reverse(new[] { 1, 2, 2, 3 }));
+            Assert.AreEqual(".moT rojaM ot lortnoc dnuorG", StandardFilters.Reverse("Ground control to Major Tom."));

```

Comment:
This is incorrect outcome according to the [docs](https://shopify.github.io/liquid/filters/reverse/):
> reverse cannot be used directly on a string
The docs do not note what SHOULD be the outcome, but a quick Liquid test shows no change and no exception.

Reply:
Its a simple change to faithfully implement the reverse filter description, but it seems odd to add code to remove potentially useful behaviour.
Should we prioritise compliance with the docs over functionality? (I’d vote for keeping string reversal).

Reply:
Changed to prevent string reversal.